package.json

```diff
@@ -18,7 +18,7 @@
     "coverage": "NODE_ENV=test nyc --reporter html npm run tape && opener coverage/index.html",
     "build": "NODE_ENV=production browserify index.js > dist/mapbox-gl-draw.js",
     "prepublish": "NODE_ENV=production browserify index.js | uglifyjs -c -m > dist/mapbox-gl-draw.js",
-    "start": "node server.js"
+    "start": "PORT=8080 node server.js"
```

Comment:
We shouldn't override what PORT can do. If we need to change the default port, lets change in server.js